website/client/src/components/tasks/column.vue

```diff
@@ -666,9 +666,10 @@ export default {
     filterByLabel (taskList, filter) {
       if (!taskList) return [];
       return taskList.filter(task => {
+        if (filter === 'all') return true;
```

Comment:
is the last filter (return !task.completed;) used somewhere now that we have the `all` one? Is it okay to change the behavior from `!task.completed` to showing all tasks?